lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1146,6 +1146,15 @@ sub create_bam_header {
     return 1;
 }
 
+sub set_bam_size {
+    my ($self, $bam_file) = @_;
+    $bam_file = $self->get_bam_path unless defined $bam_file;
```

Comment:
I don't think `get_bam_path` exists.  Both `get_bam_file` and `bam_path` do, though.